src/pages/signin.tsx

```diff
@@ -102,7 +143,15 @@ export default function Signin() {
               />
             </div>
 
-            <Button type="submit" colorScheme="primary" size="full" className="mb-3.5">
+            <div id="error_message" />
+
+            <Button
+              type="submit"
+              colorScheme="primary"
+              size="full"
+              className="mb-3.5"
+              disabled={!isValid}
```

Comment:
```suggestion
              disabled={isDirty && !isValid}
```
Take advantage of the `isDirty` to only disable the button after the user types, so it isn't invalid as soon as the page loads